rafthttp/peer.go

```diff
@@ -76,6 +76,9 @@ type Peer interface {
 	// activeSince returns the time that the connection with the
 	// peer becomes active.
 	activeSince() time.Time
+	// isActive returns true if the connection to this peer
+	// has been established
+	isActive() bool
```

Comment:
we can reuse activeSince. if it is smaller than current, it it is active, right?